x/wasm/internal/keeper/genesis.go

```diff
@@ -0,0 +1,74 @@
+package keeper
+
+import (
+	"bytes"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmwasm/wasmd/x/wasm/internal/types"
+	// authexported "github.com/cosmos/cosmos-sdk/x/auth/exported"
+	// "github.com/cosmwasm/wasmd/x/wasm/internal/types"
+)
+
+// InitGenesis sets supply information for genesis.
+//
+// CONTRACT: all types of accounts must have been already initialized/created
+func InitGenesis(ctx sdk.Context, keeper Keeper, data types.GenesisState) {
+	for id, info := range data.CodeInfos {
+		bytecode := data.CodesBytes[id]
+		newId, err := keeper.Create(ctx, info.Creator, bytecode)
+		if err != nil {
+			panic(err)
+		}
+		newInfo := keeper.GetCodeInfo(ctx, newId)
```

Comment:
What is this check for?

Reply:
Just a sanity check to make sure it registered the Code as having the same creator it did last time.